lib/jsdom/living/nodes/Node-impl.js

```diff
@@ -86,13 +90,9 @@ class NodeImpl extends EventTargetImpl {
     return domSymbolTree.parent(this);
   }
 
-  getRootNode() {
-    // ignore option for composed, because of no Shadow DOM support
-    let root;
-    for (const ancestor of domSymbolTree.ancestorsIterator(this)) {
-      root = ancestor;
-    }
-    return root;
+  getRootNode(options = {}) {
```

Comment:
The defaulting should be taken care of by the Web IDL layer. So this should just be `getRootNode(options)` and the next line should be unnecessary.